generators/app/index.js

```diff
@@ -1,114 +1,109 @@
-var generators = require('yeoman-generator');
-var _ = require('lodash');
+var generators = require('yeoman-generator')
+var _ = require('lodash')
```

Comment:
Use meaningful variable names. Let minification do the magic for you automatically.